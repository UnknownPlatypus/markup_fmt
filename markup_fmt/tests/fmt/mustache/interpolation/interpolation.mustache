Hello from {Mustache }!

{{  template}}: {{ planet    }}

These characters should not be HTML escaped: {{{  forbidden  }}}

These characters should not be HTML escaped: {{&forbidden }}

{{ person.name}}
{{ . }}
{{{ . }}}

{{> partial }}

{{> * dynamic }}
