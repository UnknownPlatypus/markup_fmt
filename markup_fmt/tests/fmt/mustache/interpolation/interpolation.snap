---
source: markup_fmt/tests/fmt.rs
---
Hello from {Mustache }!

{{template}}: {{planet}}

These characters should not be HTML escaped: {{{forbidden}}}

These characters should not be HTML escaped: {{&forbidden}}

{{person.name}}
{{.}}
{{{.}}}

{{> partial}}

{{> * dynamic}}
